app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,24 @@
+<nav class="navbar navbar-default">
+  <div class="navbar-header">
+    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+    <span class="sr-only">Toggle navigation</span>
+    <span class="icon-bar"></span>
+    <span class="icon-bar"></span>
+    <span class="icon-bar"></span>
```

Comment:
The little hamburger-type thing that people associate with opening menus.

<img width="43" alt="screen shot 2017-01-30 at 3 45 32 pm" src="https://cloud.githubusercontent.com/assets/9057935/22440903/36f211c0-e703-11e6-8ae3-aa027e7a90e5.png">